### ISSUE TITLE
physical not exported

### ISSUE THREAD

**Issue by kindlychung** [edwindj/ffbase#43]
If I import `load.ffdf, save.ffdf` from `ffbase` and use them in the R package I am developing, I got an error saying the function `physical` cannot be found. If I try to import `physical`, too, then I get an error saying that `physical` is not exported from namespace `ffbase`. 

To work around this problem, I have to add `export("physical")` in the NAMESPACE file.

**Comment by edwindj** [comment]
Thanks for reporting and suggestion. I will fix it shortly.

Best, Edwin

**Comment by edwindj** [comment]
@kindlychung 
Could you check if the latest github version solves your problem?

```
devtools::install_github('edwindj/ffbase', subdir='pkg')
```

**Comment by kindlychung** [comment]
No, I still got an error:

    Error : object ‘physical’ is not exported by 'namespace:ffbase'

**Comment by edwindj** [comment]
`physical` is a function/generic that is exported by `ff` and not `ffbase`. Is your problem solved if you import `physical` from `ff`?

**Comment by kindlychung** [comment]
Yeah, that's the weird part. Exporting `physical` from `ff` has no effect at all on this error, but exporting from ffbase solves it.
